.github/workflows/ubuntu.yml

```diff
@@ -191,6 +191,15 @@ jobs:
         batch-name: valid-v910
         os: ${{ env.os }}
 
+    - name: Run tests for time series generator tool
+      if: ${{ env.RUN_SIMPLE_TESTS == 'true' }}
+      uses: ./.github/workflows/run-tests
+      with:
+        simtest-tag: ${{steps.simtest-version.outputs.prop}}
+        batch-name: ts-generator
+        os: ${{ env.os }}
+        variant: "tsgenerator"
+
```

Comment:
revert

Reply:
Tests for thermal ts generation, why remove it ?